kubernetes/data_source_kubernetes_endpoints_test.go

```diff
@@ -0,0 +1,65 @@
+package kubernetes
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	api "k8s.io/api/core/v1"
+)
+
+func TestAccKubernetesDataSourceEndpoints_basic(t *testing.T) {
```

Comment:
Please add a `V1` suffix to the function name to align with `kind` and `apiVersion` -- TestAccKubernetesDataSourceEndpointsV1_basic.